### PR TITLE
[SC-125] September 13 spell

### PR DESCRIPTION
**Pull request by barrutko**
- ~~For some weird reason Goerli E2E tests fail with `[FAIL. Reason: 50]`. Need to investigate that.~~ We'll fix that in a separate PR.
- ~~This time these spells are exactly identical. They beg for refactoring. To be done.~~ Saving this one for later.

### CODE REVIEW

File: src/proposals/20230913/SparkEthereum_20230913.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkEthereum_20230913.sol:10]: Forum link is incorrect, correct one here: https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801

File: src/proposals/20230913/SparkGoerli_20230913.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkGoerli_20230913.sol:10]: Forum link is incorrect, correct one here: https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801

File: src/proposals/20230913/SparkEthereum_20230913.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkEthereum_20230913.sol:12]: Vote links are correct, lgtm

File: src/proposals/20230913/SparkGoerli_20230913.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkGoerli_20230913.sol:12]: Vote links are correct, lgtm

File: src/proposals/20230913/SparkEthereum_20230913.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkEthereum_20230913.sol:16]: Address is correct

File: src/proposals/20230913/SparkEthereum_20230913.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkEthereum_20230913.sol:17]: Checked interest rate strategy borrowSpread, 0.5%

File: src/proposals/20230913/SparkEthereum_20230913.t.sol
Comment (lucas-manuel) [inline src/proposals/20230913/SparkEthereum_20230913.t.sol:71]: Nice using the old strategy state for this assertion

**Review by lucas-manuel — COMMENTED** [review]
Payloads need to update the forum link, other than that I'm good to deploy.

Will do another review of deployed payloads, but logic and tests are sound.

In order to get an approval, we'll need to figure out the e2e issue.

Did the following:
- Reviewed the diff file
- Reviewed all links in payloads
- Reviewed payload logic
- Reviewed payload addresses on both goerli and ethereum
- Checked `borrowSpread` value manually on etherscan for both goerli and ethereum
- Reviewed test logic
- Reviewed test addresses

LGTM for a deployment, after forum links are updated.

**Review by hexonaut — COMMENTED** [review]
Looks good to deploy once the forum post comment is fixed. Checked:

 * Payload hardcoded addresses are good
 * DAI IRS is deployed correctly
 * Payload logic is correct
 * Test hardcoded values are correct
 * Tests look good
 * Tests are passing (except the Goerli e2e issue)
 * Goerli matches the Ethereum code with proper different addresses

File: src/proposals/20230913/SparkEthereum_20230913.t.sol
Comment (barrutko) [reply src/proposals/20230913/SparkEthereum_20230913.t.sol:71]: 🙏

**Review by hexonaut — APPROVED** [review]
Good to deploy.

**Review by lucas-manuel — DISMISSED** [review]
Good to deploy

**Review by hexonaut — APPROVED** [review]
LGTM. Sending to Maker.

**Review by hexonaut — APPROVED** [review]

**Review by lucas-manuel — APPROVED** [review]
LGTM

**Comment by hexonaut** [comment]
Let's add a Tenderly fork here too.

**Comment by hexonaut** [comment]
This has gone out so going to merge as Bartek is in transit.

**Pull request merged**
